src/arrayMethodSort.js

```diff
@@ -3,9 +3,43 @@
 /**
  * Implement method Sort
  */
+
+function compareAsString(a, b) {
```

Comment:
You could consider moving this function into the scope of sort2 function to avoid making sort2 dependent on external functions.

Reply:
I honestly think it can be the way it is.